metatron-app/metatron-discovery
# #3768 improved dashboard list scroll

### Description

improved dashboard list scroll

**Related Issue** : #3768 


### How Has This Been Tested?

1. Make sure the dashboard list keeps scrolling when you click on it.
2. Check if scrolling is maintained when returning to the browsing screen from the edit screen.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
